src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,50 @@
+package lotto.domain;
+
+import java.util.Objects;
+import java.util.Set;
+
+public class Lotto {
+
+    private static final int LOTTO_NUMBERS_SIZE = 6;
+    private static final String ERROR_LOTTO_NUMBER = "로또 번호는 6개여야 합니다.";
+
+    private final Set<LottoNumber> lotto;
+
+    public Lotto(Set<LottoNumber> lotto) {
+        validate(lotto);
+        this.lotto = lotto;
+    }
+
+    private void validate(Set<LottoNumber> lottoNumber) {
+        if (lottoNumber.size() != LOTTO_NUMBERS_SIZE) {
+            throw new IllegalArgumentException(ERROR_LOTTO_NUMBER);
+        }
+    }
```

Comment:
사소해보일 수 있지만, 아래와 같이 변수명에도 신경써주세요! 🙏
```suggestion
    private final Set<LottoNumber> lottoNumbers;

    public Lotto(Set<LottoNumber> lottoNumbers) {
        validate(lottoNumbers);
        this.lottoNumbers = lottoNumbers;
    }

    private void validate(Set<LottoNumber> lottoNumbers) {
        if (lottoNumbers.size() != LOTTO_NUMBERS_SIZE) {
            throw new IllegalArgumentException(ERROR_LOTTO_NUMBER);
        }
    }
```

Reply:
반영하였습니다 👍